src/basic/meson.build

```diff
@@ -19,6 +19,7 @@ basic_sources = files(
         'chattr-util.c',
         'conf-files.c',
         'confidential-virt.c',
+        'curl-util.c',
```

Comment:
this really should be in shared/ not basic/.

Reply:
Moved to shared/, thank you.